.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+lib
```

Comment:
Doofe Frage – Du hast es wahrscheinlich im Video erklärt: Wofür braucht man den `lib`-Ordner? `dist` ist mir klar, aber wofür `lib`?

Und sollte man diese Datei nicht eher pro Folge in das jeweilige Verzeichnis legen, und ganz oben nur die allgemeine, wo zB die `thumbs.db` und solche Sachen ausgeschlossen werden?

Reply:
Ne, das habe ich im Video tatsächlich nicht erklärt. 

Hintergrund:
Ich wollte mich hier erstmal an die Docs von Babel halten (s.h. https://babeljs.io/docs/en/usage), wo das so gemacht wird. Falls ein*e Zuschauer*in in die Babel-Docs schaut, damit sie sich da schnell zurecht findet. 

Und was meinst du mit `dieser Datei`? Das hab ich noch nicht ganz verstanden 😅